Tasks/SqlAzureDacpacDeployment/FindSqlPackagePath.ps1

```diff
@@ -441,12 +427,62 @@ function Locate-HighestVersionSqlPackageInVS()
     return $null, 0
 }
 
+function Get-VisualStudio_15_0 {
+    [CmdletBinding()]
+    param()
+
+    $visualStudioInstallDir = $null
+    try {
+        # Query for the latest 15.* version.
+        #
+        # Note, the capability is registered as VisualStudio_15.0, however the actual version
+        # may be something like 15.2.
+        Write-Verbose "Getting latest Visual Studio 15 setup instance."
+        $output = New-Object System.Text.StringBuilder
+        Invoke-VstsTool -FileName "$PSScriptRoot\vswhere.exe" -Arguments "-version [15.0,16.0) -latest -format json" -RequireExitCodeZero 2>&1 |
```

Comment:
VSWhere is not dependent on specific version of .NET framework. Have tested with 2.0